docs/eventlisteners.md

```diff
@@ -65,38 +97,48 @@ triggers:
       name: pipeline-template
 ```
 
-### ServiceAccountName
+Also, to support multi-tenant styled scenarios, where an administrator may not want all triggers to have
+the same permissions as the `EventListener`, a service account can optionally be set at the trigger level
+and used if present in place of the `EventListener` service account when creating resources:
 
-The `serviceAccountName` field is required. The ServiceAccount that the
-EventListener sink uses to create the Tekton resources. The ServiceAccount needs
-a role with the following rules:
+```yaml
+triggers:
+  - name: trigger-1
+    serviceAccount: 
+      name: trigger-1-sa
+      namespace: event-listener-namespace
+    interceptors:
+      - github:
+          eventTypes: ["pull_request"]
+    bindings:
+      - name: pipeline-binding
+      - name: message-binding
+    template:
+      name: pipeline-template
+``` 
 
-<!-- FILE: examples/role-resources/triggerbinding-roles/role.yaml -->
+*NOTE*: When employing per trigger service accounts, if the specified service account resides in a namespace that differs
+from the namespace of the `EventListener`, you'll need apply cluster level access for service accounts and secrets
```

Comment:
Even if it is on the same namespace, the EL will still need access to the `ServiceAccount`s right?

Reply:
ok I thought I saw somewhere it already have that but I'm probably mistaken

I'll make this more generic.

Reply:
Furthermore @dibyom ... aside from your point in this thread, based on our resolutions to the questions below this doc IMO will need some more beef to it one way or the other around multi-tenant, etc. ... I'll do that once we get consensus on the open questions.

Reply:
OK circled back to this and yep I was mistaken ... `ServiceAccounts` are not in the `ClusterRole` as defined at https://github.com/tektoncd/triggers/blob/master/config/200-clusterrole.yaml

What I am now thinking I'll do is update https://github.com/tektoncd/triggers/blob/master/config/200-clusterrole.yaml as part of keeping the current form of this PR with adding auth at the `EventListenerTrigger` level, and add read level access to service accounts, and nuke this portion of the doc update along with adding the overall multi-tenancy piece with pros/cons, etc.